### ISSUE TITLE
Condsider adding an HTTP/3 column

### ISSUE THREAD

**Issue by LPardue** [igrigorik/istlsfastyet.com#203]
There is an HTTP/2 column and since people are making progress on IETF QUIC & HTTP/3 interoperability it seems it might be useful to add an HTTP/3 column.

There is an argument to be made for a QUIC column too but that will likely proliferate to needing to disambiguate between gQUIC and IETF QUIC. So for simplicity sake I'd recommend not presenting that.

**Comment by igrigorik** [comment]
Agreed, adding H/3 columns makes sense to me. Willing to pull together a PR? I guess the tricky bit here would be to (ideally) canvas and gather current state on our own, prior to publishing — not a strict blocker though.

**Comment by LPardue** [comment]
Let me take a crack...

**Comment by LPardue** [comment]
It should probably be a WIP PR until such a time as HTTP/3 is finalized - then it can reflect whether an implementation has mainline support for the singular RFC version.

**Comment by igrigorik** [comment]
👍

**Comment by bazzadp** [comment]
I literally came here to suggest (and/or do) this! Great minds think alike @LPardue 😀
